lib/elastic_apm/config/options.rb

```diff
@@ -69,6 +70,11 @@ def normalize_dict(val)
           return val unless val.is_a?(String)
           Hash[val.split(/[&,]/).map { |kv| kv.split('=') }]
         end
+
+        def normalize_url(val)
+          return val unless val.is_a?(String)
```

Comment:
I know this is merged already but why this check? Wouldn't we want to `ArgumentError` if given anything but a string?

Reply:
Anything but a string as `server_url` will error somewhere else down the line.

Reply:
I was following the pattern of the other `normalize_*` methods. I think I should actually call `to_s` on it, as that is [what we were doing before](https://github.com/elastic/apm-agent-ruby/blob/master/lib/elastic_apm/config/options.rb#L45). What do you think?

Reply:
The others make sense because they need to handle for example both an array and a string value, keeping the array if given one or converting into an array if given a string. This can only be passed and used as a string – and not, say, stored as a `URI`.

`to_s` instead is good 👍

Reply:
With the current change, this would set `server_url` to a `URI` if given one, exploding later on when we append strings to it using `+`, etc...

Reply:
got it. So I'll change it to `to_s`

Reply:
https://github.com/elastic/apm-agent-ruby/pull/573